### ISSUE TITLE
Styling Select Menu

### ISSUE THREAD

**Issue by yukseltron** [sakofchit/system.css#24]
Just an idea:

Currently, the Select Menu component's appearance is based on the user's OS. 
<img width="612" alt="Screen Shot 2022-11-03 at 9 04 05 AM" src="https://user-images.githubusercontent.com/14843458/199727819-fcf5d61d-3b54-4128-90a9-7fbad38cd334.png">

It'd be nice if it looked like something closer to the Dropdown menu component
<img width="289" alt="Screen Shot 2022-11-03 at 9 05 12 AM" src="https://user-images.githubusercontent.com/14843458/199728024-497fce02-5b2f-4021-aa56-7bba8a142a30.png">

Styling it doesn't seem to be tough a task. However, accessibility is my bigger concern. The component may need a `tabindex=0` property in its HTML in order to preserve page tabbing. This is not possible to enforce using CSS alone, so docs may need to reflect the importance of having a `tabindex=0` attribute.

**Comment by sakofchit** [comment]
Agreed, I couldn't seem to figure out how to style it without needing to use JS which is why I left it as is, but if you're able to get it to work that'd be swell.

This is what I looked at before: https://www.w3schools.com/howto/howto_custom_select.asp

**Comment by Maithani-ji** [comment]
If you want simple dropdown like second picture , i think it can be achieved using css --> :hover effect .
If  you can assign it to me i can do it .
